YAXLib/YAXSerializerOfT.cs

```diff
@@ -13,33 +13,47 @@ namespace YAXLib
     ///     policy.
     ///     This class also supports serializing most of the collection classes such as the Dictionary generic class.
     /// </summary>
-    public class YAXSerializer<T> : YAXSerializer
+    public class YAXSerializer<T> : IYAXSerializer<T>
```

Comment:
We could also explicit implement `IYAXSerializer<object>`. So you cast it to that version if needed

Reply:
Yeah, this still is an issue with PRs. 
`Run .\.sonar\scanner\dotnet-sonarscanner begin /k:"yaxlib_YAXLib" /o:"yaxlib" /d:sonar.login="" /d:sonar.host.url="https://sonarcloud.io/" /d:sonar.cs.opencover.reportsPaths="**/coverage*.xml"`
No permission to read `secrets.SONAR_TOKEN`. That's why `sonar.login` is left empty.

Reply:
Hm, right, but unless you see a good reason pro `object` I would leave `IYAXSerializer<T>`.